lib/swagger-client.js

```diff
@@ -33,14 +33,14 @@ const Request = require('./request')
 
 class Root extends Component {
   _getSpec (pathname) {
-    return this.http.http({ method: 'GET', pathname })
+    const noAuthHTTP = new Request({
+      url: this.http.requestOptions.baseUrl,
+      ca: this.http.requestOptions.ca
+    })
+    return noAuthHTTP.http({ method: 'GET', pathname })
```

Comment:
What if we added a `noAuth` option to `http`?  One benefit of doing that is if we modify `Request` or add a new version and a user constructs it with some custom configuration that we wouldn't know how to replicate when we construct it here.

Thoughts?

Reply:
Thoughts on https://github.com/godaddy/kubernetes-client/pull/372/files#r242026927 ?